### DISCUSSION TITLE
Cant connect LDAP user after migration from 3.0.10.49 to 3.1.1.40

### DISCUSSION THREAD

**Discussion by periway** [nilsteampassnet/TeamPass#4009] [Q&A]
HI Nils,

First thank you for your amazing product. It's a very eleguant tools.

I have migrate my server from Debian 11 php7.4 to Debian 12 php8.2. Everything work fine.

Then i have upgrade my teampass installation from 3.0.10.49 to 3.1.1.40 and i cant connect anymore with an LDAP user. I can use only my local test account or admin account.

In apache log i see this error:
[Mon Jan 15 13:21:05.758858 2024] [php:error] [pid 8479] [client x.x.x.x:57696] PHP Fatal error:  Uncaught Error: Call to a member function isEnabled() on null in /var/www/TeamPass/vendor/teampassclasses/ldapextra/src/ActiveDirectoryExtra.php:137\nStack trace:\n#0 /var/www/TeamPass/sources/identify.php(1191): TeampassClasses\\LdapExtra\\ActiveDirectoryExtra->userIsEnabled()\n#1 /var/www/TeamPass/sources/identify.php(2289): authenticateThroughAD()\n#2 /var/www/TeamPass/sources/identify.php(328): identifyDoLDAPChecks()\n#3 /var/www/TeamPass/sources/identify.php(162): identifyUser()\n#4 /var/www/TeamPass/sources/identify.php(194): handleAuthAttempts()\n#5 {main}\n  thrown in /var/www/TeamPass/vendor/teampassclasses/ldapextra/src/ActiveDirectoryExtra.php on line 137, referer: https://teampass.ofp/index.php

Everything work fine when i comeback to my 3.0.10.49 backup.

Any idea?

Kind regards

**Comment by gonzalodeniz** [comment]
Have you found any solución?
